src/meta/meta_bulk_load_service.cpp

```diff
@@ -101,6 +112,9 @@ void bulk_load_service::on_start_bulk_load(start_bulk_load_rpc rpc)
              request.file_provider_type,
              request.remote_root_path);
 
+    remove_bulk_load_dir_on_remote_storage(app, false);
```

Comment:
As we decide to not remove app bulk load directory when the last bulk load finished, there are two following solutions:
1. remove potential app when start bulk load, then create directory immediately
2. update original create directory logic, use not-exist then create, exist then update

In current implemetation, you choose solution 1, and I prefer solution 2. Solution 2 can avoid sleep for waiting removing succeed and combine remote storage operation together. You can update function `do_start_app_bulk_load` to implement solution2.